src/unittests/f32_interval.spec.ts

```diff
@@ -552,7 +553,7 @@ g.test('ulpInterval')
 
 interface PointToIntervalCase {
   input: number;
-  expected: [number, number];
+  expected: [number, number] | F32Interval;
```

Comment:
This change is subtle. If I understand this correctly, you now apply error tolerances to the array limits, but not the interval.
A comment explaining the difference between these types is needed.
Ideally this would be in a separate change too.

Reply:
Ack, will do

Reply:
Done in https://github.com/gpuweb/cts/pull/1550

Reply:
Thanks. I had no issues with this being two CLs of the same patchset (you can review the CLs separately), so long as this is landed as a rebase (not a squash).
I'll approve this change, and you can likely abandon #1550